travis/test_server.py

```diff
@@ -155,22 +154,27 @@ def get_addons_to_check(travis_build_dir, odoo_include, odoo_exclude):
     return addons_list
 
 
-def get_test_dependencies(repo_dir, addons_list):
+def get_test_dependencies(addons_path, addons_list):
     """
     Get the list of core and external modules dependencies
     for the modules to test.
-    :param repo_dir: the directory with the modules to test
+    :param addons_path: comma separated list of addons paths
     :param addons_list: list of the modules to test
     """
     if not addons_list:
         return ['base']
     else:
-        manif_path = os.path.join(repo_dir, addons_list[0], '__openerp__.py')
-        manif = eval(open(manif_path).read())
-        return list(
-            set(manif.get('depends', []))
-            | set(get_test_dependencies(repo_dir, addons_list[1:]))
-            - set(addons_list))
+        for path in addons_path.split(','):
+            for manifest_file in manifest_files:
+                manif_path = os.path.join(path, addons_list[0],
+                                          manifest_file)
+                if not os.path.isfile(manif_path):
+                    continue
```

Comment:
Its good, but f we change `is:_module` so that it returns the manifest file name, this could could be very much simpler!